2019/5xxx/CVE-2019-5624.json

```diff
@@ -1,18 +1,127 @@
 {
-    "CVE_data_meta": {
-        "ASSIGNER": "cve@mitre.org",
-        "ID": "CVE-2019-5624",
-        "STATE": "RESERVED"
-    },
-    "data_format": "MITRE",
-    "data_type": "CVE",
-    "data_version": "4.0",
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "generator": {
+    "engine": "Vulnogram 0.0.6"
+  },
+  "CVE_data_meta": {
+    "ID": "CVE-2019-5624",
+    "ASSIGNER": "cve@rapid7.com",
+    "DATE_PUBLIC": "2019-04-24T18:00:00.000Z",
+    "TITLE": "Rapid7 Metasploit Framework Zip Import Directory Traversal",
+    "AKA": "",
+    "STATE": "PUBLIC"
+  },
+  "source": {
+    "defect": [
+      "MS-3923"
+    ],
+    "advisory": "",
```

Comment:
No idea what this field means (aside from sounding like a duplicate of CONFIRM REF value maybe). I don't see it on previous ones, so seems best to drop.

Reply:
Fixed, dropped, doesn't add much anyway.

Reply:
Sorry, I just meant the "advisory" field here, not "defect". Not a big deal for this, but including defect issue numbers seems good and fine for easier internal debug if needed.